app.py

```diff
@@ -1,39 +1,77 @@
-from flask import Flask, jsonify, redirect, render_template, request
-from flask_cors import CORS
-from lego_prediction import pricer
+from flask import Flask, render_template, request, redirect
+from flask_sqlalchemy import SQLAlchemy
+from form import UrlForm
+import shortuuid
 from werkzeug import exceptions
+import os 
+
 
 app = Flask(__name__)
-CORS(app)
+app.config['SECRET_KEY'] = '5204956a60384b5685e8ce01e34235517b576fc6a461ff13'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///site.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+@app.before_first_request
+def create_tables():
+    db.create_all()
+class UrlModel(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    url = db.Column(db.String(250), unique=True, nullable=False)
+    url_short = db.Column(db.String(100), nullable=False)
```

Comment:
url_short doesn't need to accept as many characters as you are generating a 7 character uuid